src/components/ButtonItem.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const ButtonItem = React.createClass({
+
+  render() {
+    return (
+      <button className={this.props.styles} onClick={this.props.handleClick}>
+        {this.props.text}
```

Comment:
Tips: `{this.props.children}` inneholder alt som er mellom `<ButtonItem>` og `</ButtonItem>`

Da er det mulig å lage ButtonItem på denne måten:
```
<button className={this.props.styles} onClick={this.props.handleClick}>
    {this.props.children}
</button>

<ButtonItem styles={styles.buttonStudent} handleClick={this.displayInfo}>Lærer/Veileder</ButtonItem>
```
Så blir det litt mer som å bruke en vanlig button.